pkg/operator/operator.go

```diff
@@ -154,7 +164,7 @@ func New(config operatorconfig.Config, dnsManager dns.Manager, kubeConfig *rest.
 
 	return &Operator{
 		manager: operatorManager,
-		caches:  []cache.Cache{operandCache},
+		caches:  []cache.Cache{operandCache, globalCache},
```

Comment:
Does `globalCache` supersede `operandCache`? Maybe we can remove `operandCache` in a follow-up.